jekyll/_cci2/configuration-reference.md

```diff
@@ -727,14 +742,30 @@ only | N | String, or List of Strings | Either a single branch specifier, or a l
 ignore | N | String, or List of Strings | Either a single branch specifier, or a list of branch specifiers
 {: class="table table-striped"}
 
+###### **`tags`**
+Tags can have the keys `only` and `ignore`. CircleCI will not build Git tags unless they are specified under your workflows job configuration. **Note:** This is different than how Git push branches work. When CircleCI sees an unrecognized branch, it runs.  When CircleCI sees an unrecognized tag, it is ignored. You need to add tag filters under all the jobs which you would like to run as part of Git tag.   
+
+- Any tags that match `only` will run the job.
+- Any tags that match `ignore` will not run the job.
+- If neither `only` nor `ignore` are specified then all branches will run the job.
```

Comment:
"If neither `only` nor `ignore` are specified then all branches will run the job." -> "If neither `only` nor `ignore` are specified then the job is skipped for all tags."